<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обработка данных о выставочных экспонатах.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="main_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Данные/таблицы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="table_tab">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>НИР</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableView" name="table_NIR">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="Button_add">
                <property name="text">
                 <string>Добавить строку</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button_change">
                <property name="text">
                 <string>Изменить строку</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="Button_delete">
                <property name="text">
                 <string>Удалить строку</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="comboBox_sort"/>
              </item>
              <item>
               <widget class="QPushButton" name="Button_filter">
                <property name="text">
                 <string>Фильтрация </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Группы НИР</string>
           </attribute>
           <widget class="QTableView" name="new_table_NIR">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>657</width>
              <height>393</height>
             </rect>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
           <widget class="QComboBox" name="new_table_combobox">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>181</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>ВУЗы</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableView" name="table_VUZ">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>ГРНТИ</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableView" name="table_GRNTI">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Сервис</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="show_NIR">
   <property name="text">
    <string>НИР</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="show_VUZ_2">
   <property name="text">
    <string>Вузы</string>
   </property>
  </action>
  <action name="show_grnti">
   <property name="text">
    <string>ГРНТИ</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
